templates/bootstrap/field.html

```diff
@@ -6,9 +6,15 @@
   <{% if tag %}{{ tag }}{% else %}div{% endif %} id="div_{{ field.auto_id }}" class="control-group{% if wrapper_class %} {{ wrapper_class }}{% endif %}{% if form_show_errors%}{% if field.errors %} error{% endif %}{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}">
     {% if field.label and not field|is_checkbox and form_show_labels %}
       <label for="{{ field.id_for_label }}" class="control-label {% if field.field.required %}requiredField{% endif %}">
-        {{ field.label|safe }}{% if field.field.required %}<span class="required-label">{% if field.label == 'Abstract' %}140 Characters + {% endif %}required</span>{% endif %}
-      </label>
-    {% endif %}
+      {{ field.label|safe }}
+      {% if field.field.required %}
+        <span class="required-label">
+          {% if field.name == 'abstract' %}
+            <span class="char-count">0</span> / <span>{{ field.field.max_length }}</span>
+            Characters + {% endif %}required</span>
```

Comment:
```suggestion
            Characters {% endif %}required</span>
```

Do we want the '+' there?